src/histogram-slider/types.ts

```diff
@@ -0,0 +1,21 @@
+export interface HistogramBinProps {
+    count: number;
+    minValue: number;
+}
+
+export interface HistogramSliderProps {
+    className?: string | undefined;
+    id?: string | undefined;
+    "data-testid"?: string | undefined;
+    bins: HistogramBinProps[];
+    interval: number;
+    disabled?: boolean | undefined;
+    readOnly?: boolean | undefined;
+    value?: [number, number] | undefined;
+    showRangeLabels?: boolean | undefined;
+    rangeLabelPrefix?: string | undefined;
+    rangeLabelSuffix?: string | undefined;
+    renderRangeLabel?: ((value: number) => React.ReactNode) | undefined;
+    onChange?: (value: [number, number]) => void;
```

Comment:
missing an `undefined` union type

Reply:
Might also want to add jsdocs to say the first number is the min and the second is the max